matrix/screens/bluebikes.py

```diff
@@ -1,87 +1,86 @@
-from typing import Any
 import requests
 import datetime
 from concurrent.futures import ThreadPoolExecutor
 
 from PIL import Image, ImageDraw
 
-from matrix.utils.cache import DEFAULT_IMAGE_TTL, ttl_cache
 from matrix.resources.fonts import font
-from matrix.utils.timed import timed
+from matrix.screens.screen import Screen
 
 
-@ttl_cache(seconds=59)
-@timed("bluebikes")
```

Comment:
what's the approach to keep datadog timing? i ended up removing `@timed` on things when refactoring to `Screen`, could this maybe be added to some of the methods on `BaseScreen`?

Reply:
pushed an attempt at this, lmk if you have any thoughts!